.changeset/four-cheetahs-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+"app-builder-lib": minor
```

Comment:
I realize this is a `feat`, but it seems that the `changesets-cli` CI/CD is detecting this as a major semver upgrade. For the interim, until I figure out wtf is happening there (again), can you set this to `patch`?

Reply:
You got it. Force pushed